src/login/styles.ts

```diff
@@ -61,63 +65,73 @@ export const InputArea = styled.div`
   justify-content: center;
   flex-direction: column;
 
-  @media (min-width: ${themeStyled.breakpoints.md}px) {
+  @media (min-width: ${({ theme }) => theme.breakpoints.md}px) {
     flex: 1;
   }
 `;
 
-export const InputsButtonContainer = styled.div`
+export const InputsButtonContainer = styled.form`
   display: flex;
   flex-direction: column;
-  width: 20rem;
+  width: 320px;
 
-  @media (min-width: ${themeStyled.breakpoints.md}px) {
-    width: 24rem;
+  @media (min-width: ${({ theme }) => theme.breakpoints.md}px) {
+    width: 384px;
   }
 `;
 
 export const InputContainer = styled.div`
   display: flex;
   flex-direction: column;
+  margin-top: ${({ theme }) => theme.size.m1};
 `;
 
 export const InputLabel = styled(Text).attrs({
   size: 20,
   tag: 'p',
 })`
-  color: #1d1d1d;
+  color: ${({ theme }) => theme.colors.dark['900']};
+  padding-bottom: ${({ theme }) => theme.size.m1};
 `;
 
 export const TextInput = styled.input`
   width: 100%;
-  height: 3rem;
+  height: ${({ theme }) => theme.size.m10};
   align-items: center;
   border-radius: ${({ theme }) => theme.size.m1};
-  border: 2px solid #01d5ad;
+  border: 2px solid ${({ theme }) => theme.colors.primary['700']};
+  margin-bottom: ${({ theme }) => theme.size.m4};
+  font-size: ${({ theme }) => theme.size.m5};
+  padding: ${({ theme }) => theme.size.base};
```

Comment:
Olha, aqui mesmo tem muita interpolação de função, se vc quiser, pode fazer assim também:

```suggestion
  export const TextInput = styled.input`
  width: 100%;
  align-items: center;
  ${({ theme }) => `
    height: ${theme.size.m10};
    border-radius: ${theme.size.m1};
    border: 2px solid ${theme.colors.primary['700']};
    margin-bottom: ${theme.size.m4};
    font-size: ${theme.size.m5};
    padding: ${theme.size.base};
  `}
`;
```